### PR TITLE
5.1 release blog post

### PR DESCRIPTION
**Pull request by jenweber**
This PR has the boilerplate for the 5.1 release blog post. Volunteer help is needed to fill in the details. Thanks in advance! If you have push access to this repo, feel free to push to this branch as you work, or open a PR against this branch. Reviewers are also welcome to help merge code suggestions.

## Target release window
Week of June 26

## Volunteer writers needed

Please comment on this PR if you can help out! You can look at the latest beta and changelog for each library to see what to cover.

- [x] Write ember.js section
- [x] Write ember-cli section
- [x] Write ember-data section

## Team approvals

- [x] ember.js
- [ ] ember-cli
- [x] ember-data
- [x] typescript team

### CODE REVIEW

File: content/ember-released-5-1.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+date: 2023-06-23T00:00:00.000Z
+tags:
+  - releases
+  - '2023'
+  - version-5-x
+---
+
+Today the Ember project is releasing version 5.1 of Ember.js, Ember Data, and Ember CLI.
+
+This release kicks off the 5.1 beta cycle for all sub-projects. We encourage our community (especially addon authors) to help test these beta builds and report any bugs before they are published as a final release in six weeks' time. The [ember-try](https://github.com/ember-cli/ember-try) addon is a great way to continuously test your projects against the latest Ember releases.
```
Comment (chriskrycho) [inline content/ember-released-5-1.md]:
```suggestion
This release kicks off the 5.2 beta cycle for all sub-projects. We encourage our community (especially addon authors) to help test these beta builds and report any bugs before they are published as a final release in six weeks' time. The [ember-try](https://github.com/ember-cli/ember-try) addon is a great way to continuously test your projects against the latest Ember releases.
```

File: content/ember-released-5-1.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+Ember.js is the core framework for building ambitious web applications.
+
+### Changes in Ember.js 5.1
+
+Ember.js 5.1 is an incremental, backwards compatible release of Ember with bug fixes, performance improvements, and minor deprecations.
+
+#### Bug Fixes
+
+Ember.js 5.1 introduced 0 bug fixes.
```
Comment (chriskrycho) [inline content/ember-released-5-1.md]:
```suggestion
Ember.js 5.1 introduced 1 bug fix:

- [#20461](https://github.com/emberjs/ember.js/pull/20461) Fix a cyclic module crash when using Ember Engines and Ember FastBoot.
```

File: content/ember-released-5-1.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+#### Features
+
+Ember.js 5.1 introduced 0 features.
```
Comment (chriskrycho) [inline content/ember-released-5-1.md]:
```suggestion
Ember.js 5.1 introduced 1 feature:

- [#20449](https://github.com/emberjs/ember.js/pull/20449) Publish stable TypeScript types built from Ember's source.
```

File: content/ember-released-5-1.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+- [Release Dashboard](http://emberjs.com/releases/)
+- [The Ember Release Cycle](https://blog.emberjs.com/new-ember-release-process/)
+- [The Ember Project](https://blog.emberjs.com/ember-project-at-2-0/)
+- [Ember LTS Releases](https://blog.emberjs.com/announcing-embers-first-lts/)
+
+---
+
+## Ember.js
+
+Ember.js is the core framework for building ambitious web applications.
```
Comment (chriskrycho) [inline content/ember-released-5-1.md:29]: I am working on a good, detailed write-up to go here for our "Released" TS types here—will have it up today or Monday!

Comment (chriskrycho) [reply content/ember-released-5-1.md:29]:
```suggestion
Ember.js is the core framework for building ambitious web applications.

The headlining feature of this release is stable [TypeScript](https://www.typescriptlang.org) support! 🎉 The framework now publishes TypeScript types built from Ember's own source code, with Ember's [strong Semantic Versioning commitment](https://emberjs.com/releases/) baked in—bringing benefits for both TypeScript and JavaScript users. For details, check out [the accompanying blog post][stable-ember-types].

[stable-ember-types]: https://blog.emberjs.com/stable-typescript-types-in-ember-5-1
```

Note that this means we need to merge and publish the two posts at the same time so we avoid the link being dead!

File: content/ember-released-5-1.md [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+
+The headlining feature of this release is stable [TypeScript](https://www.typescriptlang.org) support! 🎉 The framework now publishes TypeScript types built from Ember's own source code, with Ember's [strong Semantic Versioning commitment](https://emberjs.com/releases/) baked in—bringing benefits for both TypeScript and JavaScript users. For details, check out [the accompanying blog post][stable-ember-types].
```
Comment (chriskrycho) [inline content/ember-released-5-1.md]:
```suggestion
The headlining feature of this release is stable [TypeScript](https://www.typescriptlang.org) support! 🎉 The framework now publishes TypeScript types built from Ember's own source code, with Ember's [strong Semantic Versioning commitment](https://emberjs.com/releases/) baked in—bringing benefits for both TypeScript and JavaScript users. For details, a couple important caveats, and a migration guide, check out [the accompanying blog post][stable-ember-types].
```

File: content/ember-released-5-1.md [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+#### Features
+
+Ember Data 5.1 introduced 0 features.
+
+#### Deprecations
+
+Ember Data 5.1 introduced 0 deprecations.
+
+Support for the `toJSON` method on Ember Data records has been removed. It was deprecated in `3.x` and slated for removal in `4.x`.
+If your app uses this method, follow the [deprecation guide](https://deprecations.emberjs.com/ember-data/v3.x/#toc_record-toJSON).
```
Comment (runspired) [inline content/ember-released-5-1.md]: not sure why this bit is here?

File: content/ember-released-5-1.md [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+
+<!-- Block end -->
+
+For more details on changes in Ember.js 5.1, please review the [Ember.js 5.1.0 release page](https://github.com/emberjs/ember.js/releases/tag/v5.1.0).
+
+---
+
+## Ember Data
+
+Ember Data is the official data persistence library for Ember.js applications.
```
Comment (runspired) [inline content/ember-released-5-1.md]: potentially we could call out our addition of a Roadmap here. It isn't something we had in place for 5.1 specifically but is something we'll be utilizing going forward including with 5.1

**Review by jaredgalanis — APPROVED** [review]

**Pull request merged**